### DISCUSSION TITLE
Can't login, this account is blocked

### DISCUSSION THREAD

**Discussion by ongforworks** [vercel/community#5567] [Account Issue] [answered]
### Question

I signed up using github but the account got blocked, I send Vercel and email and received no-reply email said that it will take 3-4 weeks and gave me the case number  "00xxxxxxxxx" 

where can i get more information about my case?

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at registration@vercel.com from your registered email address. Thanks!
